config/config.go

```diff
@@ -175,6 +178,7 @@ type SnapshotterConfig struct {
 	ImageConfig        ImageConfig        `toml:"image"`
 	CacheManagerConfig CacheManagerConfig `toml:"cache_manager"`
 	LoggingConfig      LoggingConfig      `toml:"log"`
+	Experimental       Experimental       `toml:"experimental"`
```

Comment:
Will it move out once the feature becomes ready?

Reply:
Yes. We will move it out when it's ready. In fact, the stargz compibility feature has malfunctioned for quite a time.

Reply:
How do we ensure the CLI flags compatibility when moving it out?

Reply:
CLI does not has --enable-stargz flag now. and stargz feature has malfunctioned for a long time with stable nydusd release. So I suppose we don't have to worry about compatiblity.